kernel/nano_core/linker_higher_half-aarch64.ld

```diff
@@ -116,3 +121,7 @@ SECTIONS {
         *(.data.rel.ro.local*) *(.data.rel.ro .data.rel.ro.*)
     }
 }
+
+/* This definition must be placed at the end of the file so that the .cls,
+section is placed in the correct order. */
+__THESEUS_TLS_SIZE = SIZEOF(.tdata) + SIZEOF(.tbss);
```

Comment:
hmm, am i missing something, or is there no __THESEUS_CLS_SIZE const defined on aarch64? how does that pass linking?

Reply:
ah, nvm, i see it's gated by an arch-specific cfg.

In that case, kindly add a comment under this (and within the extern block that accesses the TLS size in `cls_offset_expr`) explaining that we don't need the CLS size on aarch64 and why.

Reply:
It does explain it, but it's still worth adding a comment such that when someone is reading or modifying the linker scripts across both architectures, they'll likely be surprised/confused by the inconsistency there. Plus, without a comment, there's no way to know that they should dig deeply into the code of that cls_macros crate.

Reply:
Yep I added a comment to the linker script, but didn't change anything in `cls_macros`.